react/components/MiniCartContent.js

```diff
@@ -62,7 +62,8 @@ class MiniCartContent extends Component {
     )
 
   getShippingCost = orderForm => {
-    const totalizer = find(propEq('id', 'Shipping'))(orderForm.totalizers)
+    const totalizers = orderForm && orderForm.totalizers || []
+    const totalizer = find(propEq('id', 'Shipping'))(totalizers)
```

Comment:
this is a small bug that I happened in a weird event and I decided to put this protection.